reference/5.1/Microsoft.PowerShell.Core/About/about_Types.ps1xml.md

```diff
@@ -433,13 +381,13 @@ the `Get-Service` cmdlet. It defines a member set named
 
 `<NoteProperty>`: Defines a property with a static value.
 
-The `<NoteProperty>` tag must have a pair of `<Name>` tags that specify
-the name of the new property and a pair of `<Value>` tags that specify
-the value of the property.
+The `<NoteProperty>` tag must have a pair of `<Name>` tags that specify the
+name of the new property and a pair of `<Value>` tags that specify the value of
+the property.
 
-For example, the following XML creates a Status property for
-directories (`System.IO.DirectoryInfo` objects). The value of the
-`Status` property is always "Success".
+For example, the following XML creates a Status property for directories
```

Comment:
Should Status here be bolded?  Also, is pluralizing directory misleading?  It's actually directory objects.

Reply:
Fixed